README.rst

```diff
@@ -0,0 +1,144 @@
+.. image:: https://circleci.com/gh/mobilityhouse/ocpp/tree/master.svg?style=svg
+   :target: https://circleci.com/gh/mobilityhouse/ocpp/tree/master
+
+.. image:: https://img.shields.io/pypi/pyversions/ocpp.svg
+   :target: https://pypi.org/project/ocpp/
+
+.. image:: https://img.shields.io/readthedocs/ocpp.svg
+   :target: https://ocpp.readthedocs.io/en/latest/
+
+OCPP
+----
+
+Python package implementing the JSON version of the Open Charge Point Protocol (OCPP). Currently
+only OCPP 1.6 is supported.
+
+Installation
+------------
+
+You can either the project install from Pypi:
+
+.. code-block:: bash
+
+   $ pip install ocpp
+
+Or clone the project and install it manually using:
+
+.. code-block:: bash
+
+   $ pip install .
+
+Quick start
+----------
+
+Below you can find examples on how to create a simple charge point as well as a charge point.
+
+.. note::
+
+   To run these examples the dependency websockets_ is required! Install it by running:
+
+   .. code-block:: bash
+
+      $ pip install websockets
+
+Central system
+~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+   import asyncio
+   import websockets
+   from datetime import datetime
+
+   from ocpp.routing import on
+   from ocpp.v16 import ChargePoint as cp
+   from ocpp.v16.enums import Action, RegistrationStatus
+   from ocpp.v16 import call_result
+
+
+   class ChargePoint(cp):
+       @on(Action.BootNotification)
```

Comment:
Cool! I see that later you explain in more depth the on decoratir. However, I didnt see any mention to the after one....shouldnt we introduce the after decorator as well?

Reply:
I don't think the quick start section is the README.md is the right place for explaining code. But maybe I should add a link to the corresponding pages of the docs.

Reply:
But I was not requesting for further explanation. I was requesting for a mention if not here somewhere, to the @after decorator as well

Reply:
Fair point. I didn't read it properly. I've updated the PR and addresses your comment.